canonical/ubuntu.com
# Refresh cloud contact us form

## Done

- Apply redesign on `__cloud-contact-us.html`
- Remove unused variable in static-forms.js
- Fix thank you page bugs

## QA

- Check out these forms:
  - [/server/contact-us](https://ubuntu-com-14079.demos.haus/server/contact-us)
  - [/openstack/contact-us](https://ubuntu-com-14079.demos.haus/openstack/contact-us)
  - [/support/contact-us?product=support-overview](https://ubuntu-com-14079.demos.haus/support/contact-us?product=support-overview)
  - [/aws/contact-us](https://ubuntu-com-14079.demos.haus/aws/contact-us)
- Check that the forms adhere to the design in [Figma file](https://www.figma.com/design/mxftgCWDJ6s3LrlaedREkP/24.10-Static-forms?node-id=1-3&t=wJYqljxJE6kXFa8h-0)
- Submit forms and check that form payload is correct on submission

## Issue / Card

Fixes [WD-13110](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/932?assignee=712020%3A55efc110-e0f6-401a-bcb0-b48a794b6dae&selectedIssue=WD-13110)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13110]: https://warthogs.atlassian.net/browse/WD-13110?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
